lib/orm/model.js

```diff
@@ -824,15 +824,18 @@ class Model {
       }
 
       models
-        .filter(associate => !associate.isSaving) // filter out models that are already being saved
-        .filter(associate => !tempAssociation.includes(associate)) // filter out models that will still be associated
+        .filter(associate =>
+            // filter out models that are already being saved
+            !associate.isSaving &&
+            // filter out models that will still be associated
+            !tempAssociation.includes(associate) &&
+            associate.hasInverseFor(association)
+        )
         .forEach(associate => {
-          if (associate.hasInverseFor(association)) {
             let inverse = associate.inverseFor(association);
 
             associate.disassociate(this, inverse);
             associate.save();
```

Comment:
Indentation must be adjusted after `if` block is removed.

Reply:
You're right. I didn't see that prettier is usable in this project. I ran prettier on the changed files and rebase + pushed the changes.

Reply:
thanks for the review @jelhan 😄 Very helpful to have other folks checking this out!

@makepanic I think I did install prettier here but still need to (a) go through and prettierify everything and (b) figure out the best way to enforce this (husky? tests?)

Reply:
We're using eslint with https://github.com/prettier/eslint-plugin-prettier which fails the linting stage if prettier has issues with our code.

Reply:
For running prettier on all project files, you could add a package.json script like:
```
    "prettier": "prettier --write \"{config,app,tests}/**/*.{js,ts}\"",
```
And adjust it to the mirage source files.